sys/include/net/gnrc/mac/mac.h

```diff
@@ -53,7 +53,7 @@ extern "C" {
  * @brief   The default queue size for transmission packets coming from higher layers
  */
 #ifndef GNRC_MAC_TX_QUEUE_SIZE
-#define GNRC_MAC_TX_QUEUE_SIZE             (8U)
+#define GNRC_MAC_TX_QUEUE_SIZE             (50U)
```

Comment:
You don't have to push every change you make ;-) https://git-scm.com/book/en/v2/Git-Tools-Stashing-and-Cleaning